### PR TITLE
tmuxinator: 0.10.0 -> 0.10.1

### PR DESCRIPTION
**Pull request by mimame**
Update tmuxinator to the last version
https://github.com/tmuxinator/tmuxinator/releases/tag/v0.10.1

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/misc/tmuxinator/default.nix [hunk 1]
```diff
@@ -8,8 +8,8 @@ buildRubyGem rec {
   inherit ruby;
   name = "${gemName}-${version}";
   gemName = "tmuxinator";
-  version = "0.10.0";
-  source.sha256 = "199pq15qknpcafw8ryb9kk1jsrwnncg6k5l9d4n0nmms4knxlqlf";
+  version = "0.10.1";
+  source.sha256 = "0rjy2glqwbz07ci0snycq19myfczd2pry2iw4g0nqsw37wclm1vi";
```
Comment (grahamc) [inline pkgs/tools/misc/tmuxinator/default.nix]: Can any of the dependencies listed below also be updated?

**Comment by grahamc** [comment]
@GrahamcOfBorg build tmuxinator

File: pkgs/tools/misc/tmuxinator/default.nix (hunk 1 above)
Comment (mimadrid) [reply pkgs/tools/misc/tmuxinator/default.nix]: Done
Sorry for the delay, I didn't see your message

**Comment by grahamc** [comment]
@GrahamcOfBorg build tmuxinator

**Comment by grahamc** [comment]
Thanks!

**Pull request merged**